Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: all clean install install-server install-cli fmt simplify check version build run
+.PHONY: all clean install install-server install-cli fmt simplify check version build run test
```

Comment:
Can you add `proto` as a dependency to the `test` rule? I got the following error when I ran `make test`:

`amp (etcd-ci)$ make test
api/server/server.go:10:2: no buildable Go source files in /Users/tony/projects/go/src/github.com/appcelerator/amp/api/rpc/service`